lib/ui/views/ib/ib_page_view.dart

```diff
@@ -183,10 +183,16 @@ class _IbPageViewState extends State<IbPageView> {
       'mark': HighlightBuilder(selectable: _selectable),
     };
 
+    // Temporary fix for bug in flutter_markdown package

```

Comment:
Is it possible to fix this issue in flutter_markdown package ? We use fltuter_markdown from [CircuitVerse/](https://github.com/CircuitVerse/packages)

Reply:
I have fixed the issue in `flutter_markdown` package in this pull request

[https://github.com/CircuitVerse/packages/pull/39](https://github.com/CircuitVerse/packages/pull/39)